compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/diagnostics/DiagnosticMessage.java

```diff
@@ -98,14 +98,26 @@ public enum DiagnosticMessage {
     ERROR_147("invalid empty record type ''{0}'' found for GraphQL input object type at field ''{1}''"),
 
     WARNING_201("invalid usage of @deprecated directive found in ''{0}''. Input object field(s) deprecation "
-                        + "is not supported by the current GraphQL spec."),
+                        + "is not supported by the current GraphQL spec"),
     WARNING_202("unable to validate ''{0}'' configuration of the GraphQL field ''{1}''. Pass a string literal to "
                         + "the ''{0}'' configuration to resolve this warning"),
     WARNING_203("unable to validate the @subgraph:Entity annotation. This could lead to an invalid generated subgraph"
-                        + " schema. Pass key-value pairs in the @subgraph:Entity annotation to resolve this warning."),
+                        + " schema. Pass key-value pairs in the @subgraph:Entity annotation to resolve this warning"),
     WARNING_204("unable to validate the @subgraph:Entity annotation. This could lead to an invalid generated subgraph"
                         + " schema. Provide a string literal or an array of string literal values for the ''{0}'' field"
-                        + " in the @subgraph:Entity annotation to resolve this warning.");
+                        + " in the @subgraph:Entity annotation to resolve this warning"),
+    WARNING_205("unable to infer default value of {0} ''{1}'' at compile time. This could potentially lead to the "
+                        + "generation of an incorrect GraphQL schema"),
+    WARNING_206(WARNING_205.getMessage() + ". Provide key-value pairs to resolve this warning"),
```

Comment:
What is the meaning of `key-value pairs` here?

Reply:
user can use short hand notation for record fields like below:
```ballerina
string sdl = "";
//....
return {sdl};
```

We can't get the default value from the default notation. User should provide a key value pair instead of short hand nottation
```ballerina
return {sdl: "sdl val"};
```